### PR TITLE
chore: remove refs to deprecated io/ioutil

### PR DESCRIPTION
**Pull request by testwill**

### CODE REVIEW

File: cmd/kbld/kbld.go [hunk 1]
```diff
@@ -18,7 +18,7 @@ import (
 func main() {
 	rand.Seed(time.Now().UTC().UnixNano())
 
-	log.SetOutput(ioutil.Discard)
+	log.SetOutput(io.Discard)
```
Comment (sethiyash) [inline cmd/kbld/kbld.go:21]: Here also should we use the `os` package only as it is already imported so we can ignore importing `io`?

File: pkg/kbld/resources/file_sources.go [hunk 2]
```diff
@@ -24,7 +24,7 @@ func NewStdinSource() StdinSource { return StdinSource{} }
 func (s StdinSource) Description() string { return "stdin" }
 
 func (s StdinSource) Bytes() ([]byte, error) {
-	return ioutil.ReadAll(os.Stdin)
+	return io.ReadAll(os.Stdin)
```
Comment (sethiyash) [inline pkg/kbld/resources/file_sources.go:27]: Here also should we use the `os` package only as it is already imported so we can ignore importing `io`?

File: pkg/kbld/resources/file_sources.go [hunk 3]
```diff
@@ -63,7 +63,7 @@ func (s HTTPFileSource) Bytes() ([]byte, error) {
 
 	defer resp.Body.Close()
 
-	result, err := ioutil.ReadAll(resp.Body)
+	result, err := io.ReadAll(resp.Body)
```
Comment (sethiyash) [inline pkg/kbld/resources/file_sources.go:66]: Here also should we use the `os` package only as it is already imported so we can ignore importing `io`?

**Comment by sethiyash** [comment]
Hi @testwill,
Thanks for raising this PR. This makes total sense to remove io/ioutil as it is deprecated as of Go 1.16.

File: pkg/kbld/resources/file_sources.go (hunk 2 above)
Comment (testwill) [reply pkg/kbld/resources/file_sources.go:27]: os.ReadAll? There is no such function

Comment (sethiyash) [reply pkg/kbld/resources/file_sources.go:27]: oh! my bad. We are good then.

**Review by sethiyash — APPROVED** [review]

**Review by joaopapereira — APPROVED** [review]
LGTM

**Pull request merged**